File: Tema2/Ejercicio08/ejercicio8.py
class ejercicio8:

    def notasAlumnos():
        
        notaAlumno = int(input("Introduce la nota del alumno al sistema: /n"))
        if notaAlumno > 0 and notaAlumno < 5:
            print("INSUFICIENTE")
        elif notaAlumno >= 5 and notaAlumno < 6:
            print("SUFICIENTE")
        elif notaAlumno >= 6 and notaAlumno < 7:
            print("BIEN")
        elif notaAlumno >= 7 and notaAlumno < 9: 
            print("NOTABLE")
        elif notaAlumno >= 9 and notaAlumno <= 10:
            print("SOBRESALIENTE")
        else:
            print("Tienes que introducir un numero correcto")

File: Tema2/Ejercicio08/ejercicio8.java
import java.util.Scanner;

public class ejercicio8 {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.println("Introduce tu nota al sistema: ");
        int notaUsuario = scanner.nextInt();

        if ((notaUsuario >= 0) && (notaUsuario < 5)){
            System.out.println("La nota se considera: INSUFICIENTE");
        }else if ((notaUsuario >= 5) && (notaUsuario < 6)) {
            System.out.println("La nota se considera: SUFICIENTE");
        }else if ((notaUsuario >= 6) && (notaUsuario < 7)) {
            System.out.println("La nota se considera: BIEN");
        }else if ((notaUsuario >= 7) && (notaUsuario < 9)) {
            System.out.println("La nota se considera: NOTABLE");
        }else if ((notaUsuario >= 9) && (notaUsuario <= 10)) {
            System.out.println("La nota se considera: SOBRESALIENTE");
        }else {
            System.out.println("Tienes que introducir la nota correctamente (0-10)");
        }
        scanner.close();
    }
}
